src/main/webapp/scripts/perspective.js

```diff
@@ -0,0 +1,22 @@
+function getToxicity() {
+  API_KEY = '';
+  const commentText = document.getElementById("comment-text").value;
+  const commentObject = {
+    'comment': {'text': commentText},
+    'requestedAttributes': {'TOXICITY': {}},
+    'languages': ['en']
+  };
+  const json = JSON.stringify(commentObject);
+  const request = new Request(
+    `https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=${API_KEY}`,
+    {method: 'POST', body: json}
+  );
+  fetch(request)
+    .then(response => response.text())
+    .then((responseText) => {
+      const data = JSON.parse(responseText);
+      const score = data.attributeScores.TOXICITY.summaryScore.value;
+      containerDOM = document.getElementById('toxicity-container');
```

Comment:
nit: use `const` by default

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let